.npmignore

```diff
@@ -1,3 +1,4 @@
 # Exclude everything but the contents of the dist directory.
 **/*
 !dist/**
+!src/css/**
```

Comment:
do we want to ignore the font folder?
If so, add `src/css/font/**` on the following line.